backstory/src/main/java/com/google/sps/perspective/PerspectiveManager.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.perspective;
+
+/**
+ * Handles analysis of stories for content and provides a recommendation on
+ * the "best" appropriate story (if multiple).
+ */
+interface PerspectiveManager {
+  /**
+   * Analyzes the passed-in story and returns PerspectiveDecision
+   * containing decision based on analysis.
+   *
+   * @param story The story to be analyzed
+   * @return An object describing the recommendation resulting from the analysis.
+   */
+  public PerspectiveDecision getDecision(String story);
```

Comment:
@annemariecabs thinking about this more, `get` is often used for property access, so might connote fast access. Maybe consider renaming to `generateDecision()` or something else that indicates that some processing is required?

Reply:
Done